cryptosigner/cryptosigner_test.go

```diff
@@ -26,12 +26,13 @@ import (
 	"fmt"
 	"testing"
 
+	btcec "github.com/decred/dcrd/dcrec/secp256k1/v2"
```

Comment:
Rename this import? Since it's not `btcec` anymore, upstream package name is `secp256k1`.